contracts/test-helpers/Helper_PrecompileCaller.sol

```diff
@@ -48,5 +48,6 @@ contract Helper_PrecompileCaller is Helper_SimpleProxy {
         )
     {
         callPrecompile(_precompile, _data);
+        return msg.sender;
```

Comment:
I have to double check this one.

Reply:
@ben-chain would be good to get your eyes on this, not sure where/if this is used. But I think `callPrecompile` does a low-level `return` inside of assembly so this return statement won't do anything (I think?)

Reply:
Yeah, this return value being added definitely goes unused.  @vcshih if the compiler shuts up with just `return;`, we can do that; otherwise `return address(0); // unused` or something feels like the way to go.

Reply:
Agree with `return address(0); // unused: silence compiler` or similar.

Reply:
Makes sense to me :+1: Implemented in the most recent commit